### ISSUE TITLE
Complete Vaccination Section

### ISSUE THREAD

**Issue by nathanbunker** [immregistries/mqe#65]
Need to write this out. Need to finish vision of vaccination view.

**Comment by michellefbarber** [comment]
Duplicate of #75 
Integrate new Vaccination XML into UI report
The vaccination information needs to be displayed. - complete after #68

**Comment by nathanbunker** [comment]
Finished getting data right. The format now needs to be tweaked:

![image](https://user-images.githubusercontent.com/122847/48512999-f4938300-e817-11e8-89d4-0ae66e5820b7.png)
